jmh/src/test/scala/magnolify/jmh/MagnolifyBench.scala

```diff
@@ -157,7 +160,152 @@ class ExampleBench {
   private val exampleNested = implicitly[Arbitrary[ExampleNested]].arbitrary(prms, seed).get
   private val example = exampleType.to(exampleNested).build()
   @Benchmark def exampleTo: Example.Builder = exampleType.to(exampleNested)
-  @Benchmark def exampleFrom: ExampleNested = exampleType.from(example)
+  @Benchmark def exampleFrom: ExampleNested =
+    exampleType.from(example.getFeatures.getFeatureMap.asScala.toMap)
```

Comment:
this needed to be updated from the ExampleType rework

Reply:
IMHO we should bench the `apply` methods

Reply:
Yeah, that would be good! I'll do that in a followup PR